### PR TITLE
Fix error caused by a blockquote inside a p tag on About page

### PR DESCRIPTION
**Pull request by ggangix**
## Description
There was an error on the About page caused by a blockquote tag inside a p tag, so I have replaced the p tag with a div with the same style as a paragraph.

## Related Issue

#23 

## Motivation and Context

Fix the error on the about page.

## How Has This Been Tested?

I just tested on the browser.

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by braidn** [comment]
Thanks for looking into this @ggangix , will look at it in a bit and get this merged.

File: styles.scss [hunk 1]
```diff
@@ -43,6 +43,13 @@ $footer-background-color: $footer-background;
   }
 }
 
+// About page
+
+.content > .leftcolumn > .paragraph {
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
```
Comment (SchaeStewart) [inline styles.scss]: Remove this.

File: pages/about.js [hunk 2]
```diff
@@ -7,11 +7,10 @@ const About = () => (
     <p className="title" >About</p>
     <div className="content">
       <div className="leftcolumn">
-        <p>
+        <div className="paragraph">
           The Code for America mission is, in part, to
           <blockquote>build open source technology and organize a network of people dedicated to making government services simple, effective, and easy to use.</blockquote>
-        </p>
-
+        </div>
```
Comment (SchaeStewart) [inline pages/about.js]: Instead of creating a new class and using a div, try just moving the `blockquote` outside of the `p` tag

**Review by SchaeStewart — CHANGES_REQUESTED** [review]
Thank you so much for contributing! I've left a few comments and I'm asking that you make a few changes before I merge your PR. Let me know if you have any questions

**Comment by ggangix** [comment]
Thanks for the feedback, the solution was more simple :)

**Comment by SchaeStewart** [comment]
Thanks!

**Pull request merged**